common/src/main/resources/tachyon-default.properties

```diff
@@ -13,6 +13,7 @@
 
 # Default configuration properties for Tachyon
 # All the properties except Build properties are override-able via tachyon-site.properties file.
+# Setting a property to -1 denotes that this property is not set but will use system default value.
```

Comment:
O, I guess I was asking, do all properties in this file have the ability to set to "-1", meaning it will take some system default value? It looks like only the 3 properties (backlog, buffer.send, buffer.receive) has the ability to set to -1 to mean "use the system default value". I don't think this comment applies to every int property in this file. For example, if I set "tachyon.worker.to.master.heartbeat.interval.ms" to -1, will the code set that value to some system default value?

If only those 3 new properties can be set to -1, I think the comment should only apply to those, and only be shown above each of those 3 properties.

Let me know if I am not understanding this correctly.